Formula/openjpeg.rb

```diff
@@ -1,12 +1,10 @@
 class Openjpeg < Formula
   desc "Library for JPEG-2000 image manipulation"
-  homepage "http://www.openjpeg.org"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/o/openjpeg/openjpeg_1.5.2.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/o/openjpeg/openjpeg_1.5.2.orig.tar.gz"
-  sha256 "aef498a293b4e75fa1ca8e367c3f32ed08e028d3557b069bf8584d0c1346026d"
-  revision 1
+  homepage "http://www.openjpeg.org/"
+  url "https://github.com/uclouvain/openjpeg/archive/v2.1.1.tar.gz"
+  sha256 "82c27f47fc7219e2ed5537ac69545bf15ed8c6ba8e6e1e529f89f7356506dbaa"
 
-  head "https://github.com/uclouvain/openjpeg.git", :branch => "openjpeg-1.5"
+  head "https://github.com/uclouvain/openjpeg.git"
```

Comment:
I inadvertently removed the `head` from the formula. Pointing it to `master` makes the most sense bacause `:branch => "openjpeg-2.1"` would merely build the most recent RB-2.1 point release.